photutils/psf/tests/test_epsf_stars.py

```diff
@@ -57,3 +59,23 @@ def test_extract_stars_inputs(self):
 
         with pytest.raises(ValueError):
             extract_stars([self.nddata, self.nddata], self.stars_tbl)
+
+
```

Comment:
```suggestion

@pytest.mark.skipif('not HAS_SCIPY')
```

Reply:
This is a function following a class.  I think 2 spaces separating them is correct (according to PEP8).

Reply:
nvm, somehow I missed the scipy skip!  sorry!

Reply:
It happens. I was just going through my notifications looking for quick fixes 😅